apps/test-viewer/src/components/repositories-tree/RepositoriesTree.tsx

```diff
@@ -3,19 +3,17 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 
-import { useState } from "react";
 import { Flex, ProgressRadial, Text } from "@itwin/itwinui-react";
-import { TreeRenderer, UnifiedSelectionProvider, useUnifiedSelectionTree } from "@itwin/presentation-hierarchies-react";
-import { TreeWidget } from "../../../TreeWidget";
-import { Delayed } from "../common/components/Delayed";
-import { ProgressOverlay } from "../common/components/ProgressOverlay";
+import { TreeRenderer, useUnifiedSelectionTree } from "@itwin/presentation-hierarchies-react";
+import { Delayed } from "../../../../../packages/itwin/tree-widget/src/components/trees/common/components/Delayed";
+import { ProgressOverlay } from "../../../../../packages/itwin/tree-widget/src/components/trees/common/components/ProgressOverlay";
+import { TreeWidget } from "../../../../../packages/itwin/tree-widget/src/TreeWidget";
```

Comment:
the app should not import internal stuff from tree widget

Reply:
changed